app/views/downloads/show.html.erb

```diff
@@ -7,7 +7,7 @@
     <% end %>
   <% # Use elsif so we don't error when there are no chapters. %>
```

Comment:
While you're here, could you clean up the ERB comment here so GitHub doesn't get confused?
```suggestion
  <%# Use elsif so we don't error when there are no chapters. %>
```